src/Storages/Distributed/DirectoryMonitor.h

```diff
@@ -65,12 +65,13 @@ class StorageDistributedDirectoryMonitor
 private:
     void run();
 
-    std::map<UInt64, std::string> getFiles();
+    std::map<UInt64, std::string> getFiles() const;
```

Comment:
What is a purpose of making this `const` + many `mutables`? I don't see if this method is called anywhere by const object.

Reply:
Indeed, this is not required anymore (it was required before, when it was used for `getStatus()`)